helpers/api_helper/json_api_response.go

```diff
@@ -18,13 +20,25 @@ func (e *ErrorResponseObject) NewErrorResponse() gin.H {
 }
 
 type DataResponseObject struct {
-	ID         string      `json:"id"`
-	Type       string      `json:"type"`
-	Attributes interface{} `json:"attributes"`
+	ID            string      `json:"id,omitempty"`
```

Comment:
@gutakk Why do we need to omit empty here?

Reply:
🤔 Yeah we do not need omit empty here as it is required field. Fixed in abbd93ca1446c9bd43282f3e8b6d3d4026bb0f9f